lib/hanami/cli/generators/gem/app/action.erb

```diff
@@ -5,5 +5,6 @@ require "hanami/action"
 
 module <%= camelized_app_name %>
   class Action < Hanami::Action
+    include Dry::Monads[:result]
```

Comment:
```suggestion
    # For pattern matching on operation results
    include Dry::Monads[:result]
```

I like that you've included this!

I suggest we include a little explainer to make it clear why we're including this.

Reply:
For consistency, can you please update the slice generator to include this too?

The difference should be that the slice generator should check to see if dry-monads is bundled before it includes this line. If the user chooses to remove dry-operation from their `Gemfile`, then dry-monads will disappear and the `include` would then result in an error.

Reply:
I did this for the slice, but the slice actions inherit from app/action.rb so it doesn't do anything. I kept it in since I might be missing something, but I think we can remove it?

I also had to add a line to require dry/monads

Reply:
@cllns Ah yeah, sorry, that's my mistake. With slice actions inheriting from an app action, we don't need to put anything additional into the slice actions.